### PR TITLE
Upgrade translation of the empty Posts list page

### PR DESCRIPTION
**Pull request by oshliaer**
I'd like to add some translations. It's not the best but it works fine.

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -13,8 +13,8 @@
 	{{- if eq $paginator.TotalNumberOfElements 0 }}
 		<div class="warning">
 			<svg class="warning__icon icon icon-files" viewBox="0 0 384 384" width="96" height="96" fill="#ddd"><path d="m368 64h-224-16v16 288 16h16 224 16v-16-288-16zm-16 288h-192v-256h192zm-320-320h192v16h32v-32-16h-16-224-16v16 288 16h16 96v-32h-80zm144 272h160v-32h-160zm0-64h160v-32h-160zm0-64h160v-32h-160zm-128 64h64v-32h-64zm0-64h64v-32h-64zm0-64h64v-32h-64z"/></svg>
-			<h3 class="warning__headline">You don't have any posts yet!</h3>
-			<p class="warning__text">As posts are added in <b>{{ replaceRE ",([^,]*)$" " or $1" (delimit (apply $postSections "printf" "content/%s" ".") ", ") }}</b> folder{{ if gt (len $postSections) 1 }}s{{ end }}, they'll appear here.</p>
+			<h3 class="warning__headline">{{ T "posts_list_warning_headline" }}!</h3>
+			<p class="warning__text">{{ T "posts_list_warning_text_l" }} <b>{{ replaceRE ",([^,]*)$" " or $1" (delimit (apply $postSections "printf" "content/%s" ".") ", ") }}</b> {{ T "posts_list_warning_text_r" (len $postSections) }}.</p>
```
[the remark this answers is not in the document]
Comment (Vimux) [reply layouts/index.html]: Can you reproduce the issue?

You can change exampleSite/config.toml (`defaultContentLanguage = "ru"`) and temporary move all .md files from `content/post` folder. After that we can test it.

https://deploy-preview-95--mainroad.netlify.com

File: i18n/en.yaml [hunk 2]
```diff
@@ -5,6 +5,18 @@
 - id: menu_label
   translation: Menu
 
+# Posts list
```
[the remark this answers is not in the document]
Comment (oshliaer) [reply i18n/en.yaml]: ba052bb

File: i18n/en.yaml [hunk 3]
```diff
@@ -5,6 +5,18 @@
...
+- id: posts_list_warning_headline
```
[the remark this answers is not in the document]
Comment (oshliaer) [reply i18n/en.yaml]: aabb85f

File: layouts/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (oshliaer) [reply layouts/index.html]: It seems it's not yada yada. If I try `fr`, `de` or `en` translation it works fine but not for `ru`. Stay tuned.

[the remark this answers is not in the document]
Comment (oshliaer) [reply layouts/index.html]: The topic on Hugo's forum https://discourse.gohugo.io/t/translation-of-the-plural-and-zero-in-russian-returns-the-name-of-the-variable/14995

[the remark this answers is not in the document]
Comment (Vimux) [reply layouts/index.html]: It shouldn't work like that. Please see [Unicode plural rules for Russian lang](https://www.unicode.org/cldr/charts/28/supplemental/language_plural_rules.html#ru) carefully.
You can not use `one`-`other` or similar i18n things, not in that way.

My suggestion:

* Rewrite Russian sentence keeping the meaning the same.

Can you do it?

[the remark this answers is not in the document]
Comment (oshliaer) [reply layouts/index.html]: As can I see it's a bug. Other languages work fine. I changed `ru` to `rus` and it works. If this is a bad option please tell me. I will fix it.

**Comment by Vimux** [comment]
`rus` is wrong lang code. So, It 'works' because Unicode doesn't have pluralization rules for `rus`.

REF: https://www.unicode.org/cldr/charts/28/supplemental/language_plural_rules.html#ru

**Comment by oshliaer** [comment]
@Vimux , this is a very clear explanation. Thank you. Let me return all to `ru`.

But why does Ukrainian (ua) work, but not Russian (ru)? No one can give me the answer to this. How can I debug of pluralization rules? Is it a Hugo or i18n lib?

**Comment by Vimux** [comment]
For the same reason as above. Use `uk` for Ukrainian.

REF: https://www.unicode.org/cldr/charts/28/supplemental/language_plural_rules.html#uk

**Pull request closed without merging**